internal/policy/commit/commit.go

```diff
@@ -33,6 +33,10 @@ type Commit struct {
 	RequireCommitBody bool `mapstructure:"requireCommitBody"`
 	// Conventional is the user specified settings for conventional commits.
 	Conventional *Conventional `mapstructure:"conventional"`
+	// HeaderCase is the case that the first word of the header must have ("upper" or "lower").
+	HeaderCase string `mapstructure:"headerCase"`
+	// HeaderInvalidLastCharacters is a string containing all invalid last characters for the header.
+	HeaderInvalidLastCharacters string `mapstructure:"headerInvalidLastCharacters"`
```

Comment:
What do you think of something like:

```
header:
  case: ...
  headerInvalidLastCharacters:
```

to keep all the header related settings together.

Reply:
We could have `body` and `footer` settings later on, and they could follow the same design.

Reply:
That makes a lot of sense; I'll work on it! Should the current `headerLength` and `imperative` also be moved there?